test/Broadway/EventSourcing/AbstractEventSourcingRepositoryTest.php

```diff
@@ -146,7 +146,7 @@ public function it_calls_the_event_stream_decorators_with_the_correct_arguments(
         $lastCall = $this->eventStreamDecorator->getLastCall();
 
         $this->assertEquals($aggregate->getAggregateRootId(), $lastCall['aggregateIdentifier']);
-        $this->assertEquals('\\' . get_class($aggregate), $lastCall['aggregateType']);
+        $this->assertEquals(get_class($aggregate), $this->stripLeadingSlash($lastCall['aggregateType']));
```

Comment:
so you're saying https://github.com/broadway/broadway/blob/master/src/Broadway/EventSourcing/EventSourcingRepository.php#L51 allows classnames with slashes and without?
This is validated by assertExtendsEventSourcedAggregateRoot  -> Assert::subclassOf which in turn calls PHP's is_subclass_of function in https://github.com/beberlei/assert/blob/master/lib/Assert/Assertion.php#L1306

wouldn't it be easier to just use assertTrue(is_subclass_of($aggregate, $lastCall['aggregateType'])) ?
or maybe even assertInstanceOf($lastCall['aggregateType'], $aggregate) ?